Taskfile.yml

```diff
@@ -35,12 +35,49 @@ tasks:
       - task: terraform-command
         vars: { TF_COMMAND: 'apply', TF_CMD_DIR: "{{.USER_WORKING_DIR}}" }
 
+  apply-demo:
+    desc: Run "terraform apply" to generate a demo account
+    dir: "./demo"
+    requires:
+      vars: [PAYLOAD]
+    env:
+      OPSLEVEL_API_TOKEN:
+        sh: echo "$PAYLOAD" | jq -r '.api_token'
+      TF_VAR_account_token:
+        sh: echo "$PAYLOAD" | jq -r '.api_token'
+      TF_VAR_user_email:
+        sh: echo "$PAYLOAD" | jq -r '.user_email'
+    cmds:
+      - "terraform init --upgrade"
+      - "terraform apply"
```

Comment:
I couldn't figure out how to get this working with the sub tasks.  Also i wanted the `apply` to prompt for approval